pkgs/development/python-modules/triton/default.nix

```diff
@@ -178,7 +178,7 @@ buildPythonPackage rec {
 
   preCheck = ''
     # build/temp* refers to build_ext.build_temp (looked up in the build logs)
-    (cd /build/source/python/build/temp* ; ctest)
+    (cd ./build/temp* ; ctest)
```

Comment:
Have you verified that this actually works on locations other than /build and isn't just relative to `/`?

Reply:
i based it on 

https://github.com/NixOS/nixpkgs/blob/58df299b0de456c0019c3fd465eca1556f2045fe/pkgs/development/python-modules/triton/default.nix#L156